matplotcheck/autograde.py

```diff
@@ -0,0 +1,99 @@
+"""
+matplotcheck.autograde
+======================
+
+Wrapper functions that run and track the passing status of
+matplotcheck functions and provide formatted results of tests.
+
+"""
+
+
+def run_test(
+    func,
+    points,
+    *args,
```

Comment:
@kcranston  should args come after the correct_message and error message parameters in the function definition?
or is it better to break them up as they are here?